docs/en/for-website-administrators/managing-roles-and-permissions.md

```diff
@@ -20,13 +20,13 @@ Learn how to restrict access by IP address
 
 ## Explaining security groups and roles
 
-Instead of assigning individual permissions to access, create, edit, or delete content per user, you can use Security Groups and Roles to organise what accounts have which permissions.  
+Instead of assigning permissions to access, create, edit, or delete content, one user at a time, you can assign these permissions to an entire group of users at once with Security Groups and Roles.  
 
 Roles are collections of permissions.  For example, you might create an editor role to give a group read/write access to all content, or a "contributor" role who has the right to add content to the CMS but not to publish it, or a "spectator" role which gives a person a right to view the backend of the CMS, but not any ability to edit it.
 
-Security groups are collections of users, and whatever permissions they have apply to a subset of pages. So, for example, the marketing team could have access to the parts of the website dealing with marketing, and the development team could have access to the parts of the website dealing with development.
+Security groups are collections of users. Whatever permissions each user in a group has, these permissions apply to pages accessible by the group. So, for example, the marketing team could have access to the parts of the website dealing with marketing (some as editors, some as contributors, and some as spectators), and the development team could have access to the parts of the website dealing with development (some as editors, some as contributors, and some as spectators).
```

Comment:
Absolutely :) Although, some reviewers don't do much of anything and should be considered passive spectators! Changed.